x/gov/client/testutil/grpc.go

```diff
@@ -357,9 +357,9 @@ func (s *IntegrationTestSuite) TestGetParamsGRPC() {
 	val := s.network.Validators[0]
 
 	params := v1.DefaultParams()
-	dp := v1.NewDepositParams(params.MinDeposit, params.MaxDepositPeriod)
-	vp := v1.NewVotingParams(params.VotingPeriod)
-	tp := v1.NewTallyParams(params.Quorum, params.Threshold, params.VetoThreshold)
+	dp := v1.NewDepositParams(params.MinDeposit, params.MaxDepositPeriod)          //nolint:staticcheck // TODO: refactor to only refer to Params
```

Comment:
why the todos

Reply:
wasn't sure how to fix-- also if to fix.

Is fix simply getting rid of everything that works like params.Thingy ?

Reply:
(or, is it just wrong, and the todos aren't needed?)